app/assets/javascripts/theme_colors.js

```diff
@@ -22,6 +22,9 @@ export const themes = {
     sharp_line: "#e5e5e5",
     strings_color: "var(--severe_error)",
     sub_menu: "var(--primary_one)",
+    submission_download_icon_fill: "var(--sharp_line)",
```

Comment:
As above, rename to something like `file_download_icon_fill`. Same for the other constants you added.